values.yaml

```diff
@@ -116,6 +116,7 @@ georchestra:
     gateway:
       enabled: false
       docker_image: georchestra/gateway:latest
+      java_tool_options: "-Dgeorchestra.datadir=/etc/georchestra -Dspring.config.location=${georchestra.datadir}/gateway/application.yaml -Dspring.profiles.active=docker"
```

Comment:
Could you put this parameter under a new section called "environment"?

Like so:
```
environment:
  JAVA_TOOL_OPTIONS: "-Dgeorchestra.datadir=/etc/georchestra -Dspring.config.location=${georchestra.datadir}/gateway/application.yaml -Dspring.profiles.active=docker"
```

This way it's categorized inside a specific section.

This makes me realized that we are missing the special `extra_environment: []` too.